src/libraries/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -189,6 +190,7 @@ public void Abort(object? stateInfo)
             throw new PlatformNotSupportedException(SR.PlatformNotSupported_ThreadAbort);
         }
 
+        [UnsupportedOSPlatform("browser")]
         public static void ResetAbort()
```

Comment:
since this throws PNSE everywhere, is there a way we can mark it as such?

Reply:
I wonder if its better to stick with the convention of marking it Obsolete. I wasn't sure if a docs issue should be linked in the parameters of `Obsolete`. It looks like it was brought up in https://github.com/dotnet/docs/issues/3825. 

Another option that was brought up was to mark this as `[UnsupportedOSPlatform("none")]`

Reply:
@jeffhandley @buyaa-n What do y'all think about `[UnsupportedOSPlatform("none")]`?

Reply:
For the analyzer that would be treated like unsupported on a platform named "none", there is no generic naming for all platforms CC @terrajobst

Reply:
`[Obsolete]` would be a better approach than to fake that with an `[UnsupportedOSPlatform]` or `[SupportedOSPlatform]` attribute.  I recommend we omit this annotation here and file a 6.0.0 issue to mark this API as Obsolete.

Reply:
I was looking at a stale page a moment ago.  I think we'll have to wait to mark this as `Obsolete` until 6.0.0 since it's technically a breaking change.  We'll want to consider using a custom diagnostic id for the obsoletion too and add it to the list of obsoletions, etc.

Reply:
I agree with Jeff. We shouldn't annotate this API. We should obsolete, but not for .NET 5.